src/sections/fragenAntworten.html

```diff
@@ -0,0 +1,99 @@
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+            </div>
+            
+            <div class=" grid gap-16 sm:mt-10 sm:flex sm:gap-10 md:gap-24">
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+            </div>
+            
+            <div class=" grid gap-16 sm:mt-10 sm:flex sm:gap-10 md:gap-24">
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+                <div>
+                    <div class="flex justify-between gap-16 lg:gap-36 2xl:gap-56 px-2">
+                        <p class="font-railwayBold">Welche Frage steht hier?</p>
+                        <p class="font-railwayBold text-2xl md:text-4xl">+</p>
+                    </div>
+                    <div class="h-1 w-full bg-green"></div>
+                </div>
+            </div> 
+        </div>
+    
+        <div class="mt-20 lg:mt-36 xl::mt-44 2xl:mt-64 pb-36">
+            <h1 class="ml-5 md:ml-16 lg:ml-28 xl:ml-36 font-railwayBold text-base sm:text-xl md:text-2xl lg:text-3xl">Nehmen Sie Kontakt zu uns auf!</h1>
+            <a href="#">
+                <div class="flex items-center gap-2 sm:gap-5 bg-yellow-500 ml-4 md:ml-12 lg:ml-24 xl:ml-32 pl-3 sm:px-7 py-2 lg:py-3 w-56 sm:w-72 md:w-80 rounded-full mt-2 sm:mt-4 md:mt-8">
+                    <img src="/images/gmail_logo_black.svg" alt="Gmail Logo in black">
```

Comment:
Good job here! 🙌🏽 You've used a relative path to reference the image, and the reason is that relative paths are often more readable and easier to maintain, especially when working with files and directories within the same project. I would recommend that you follow the same approach for other images unless it causes any issues during deployment.